tests/test_logging.py

```diff
@@ -94,3 +97,240 @@ def test_base_json_formatter_contains_service_id(tmpdir):
     service_id_filter = logging.ServiceIdFilter()
     assert json.loads(logging.BaseJSONFormatter().format(record))["message"] == "message to log"
     assert service_id_filter.filter(record).service_id == "no-service-id"
+
+
+@pytest.mark.parametrize(
+    "status_code,expected_after_level",
+    (
+        (200, builtin_logging.INFO),
+        (201, builtin_logging.INFO),
```

Comment:
Very optional: Could put in a 3xx if you want a bit more test coverage (but equally fine to not)